pyproject.toml

```diff
@@ -9,11 +9,11 @@
 
 [build-system]
 requires = [
-    "wheel",
-    "setuptools",
-    "Cython>=0.29.18",
-    "pybind11>=2.4.3",
-    "pythran>=0.9.11",
+    "wheel<0.37.0",
+    "setuptools<58.0.0",
+    "Cython>=0.29.18,<3.0",
+    "pybind11>=2.4.3,<2.7.0",
+    "pythran==0.9.11",
```

Comment:
Pinning with `==` to ensure we get that exact version, given our limited experience with it I don't want to say `>=` which can give not-yet-released bugfix versions.